js/ui/collection/ui.collection_widget.base.js

```diff
@@ -539,6 +539,12 @@ var CollectionWidget = Widget.inherit({
         if(!this.constructor.ItemClass.getInstance($item).setDataField(property, value)) {
             this._refreshItem($item, item);
         }
+
+        const isDisabling = property === 'disabled' && value;
+
+        if(isDisabling && $item.eq(this.option('focusedElement'))) {
```

Comment:
It looks like `eq` should be replaced by `is` here.

Reply:
Good note!